StarkInfra/StarkInfra/Event/Event.cs

```diff
@@ -22,7 +22,7 @@ namespace StarkInfra
     ///     <item>Created [DateTime]: creation datetime for the notification event. ex: new DateTime(2020, 3, 10, 10, 30, 0, 0)</item>
     ///     <item>IsDelivered [bool]: true if the Event has been successfully delivered to the user url. ex: False</item>
     ///     <item>Subscription [string]: service that triggered this event. ex: "pix-request.in", "pix-request.out"</item>
```

Comment:
No parâmetro **Subscription** deveria ser: `service that triggered this Event. Options: "pix-request.in", "pix-request.out", "pix-reversal.in", "pix-reversal.out", "pix-key", "pix-claim", "pix-infraction", "pix-chargeback", "issuing-card", "issuing-invoice", "issuing-purchase", "credit-note"`